gestion_base_adresse/install/sql/adresse/10_FUNCTION.sql

```diff
@@ -354,8 +359,8 @@ BEGIN
             RETURN NULL;
         END IF;
     ELSE
-        -- Sinon on modifie valide et on enregistre
-        NEW.valide = False;
+        -- Sinon on modifie a_valider et on enregistre
+        NEW.a_valider = True;
```

Comment:
```suggestion
        -- Sinon on modifie a_valider et on enregistre
        NEW.valide = False;
```
Il semblerait que ta base de données ne soit pas à jour.